torch/_inductor/fx_utils.py

```diff
@@ -126,32 +146,22 @@ def is_aten_node(node):
             if not is_aten_node(node):
```

Comment:
what nodes is this one filtering out?

Reply:
Actually I think this is really bad. IIRC when an op has multiple outputs it returns a tuple then calls `operator.getitem` to extract individual tensors. I believe we also use opeartors like mul and add on symints so this won't allow propagation through symbolic computations.